README.md

```diff
@@ -10,23 +10,68 @@ code in this repository extends an integration layer provided by [DrupalExtensio
 [![Latest Stable Version](https://poser.pugx.org/drupal/tqextension/v/stable?format=flat-square)](https://packagist.org/packages/drupal/tqextension)
 [![License](https://img.shields.io/badge/license-MIT-brightgreen.svg?style=flat-square)](https://packagist.org/packages/drupal/tqextension)
 
+## Requirements
+
+* composer
```

Comment:
Calling out dependencies is more common than showing steps for installing them. If the user needs to install a dependency they can refer to the documentation for that library.